docs/build/configure/_index.md

```diff
@@ -264,6 +264,123 @@ The corresponding raw JSON looks like this:
 
 {{% /expand %}}
 
+## Webhooks
+
+Webhooks allow you to trigger actions when certain types of data are sent from your machine to the cloud.
+For example, you can configure a webhook to send you a phone notification when your robot's battery drops below 10%.
```

Comment:
```suggestion
For example, you can configure a webhook to send a phone notification when your robot's battery drops below 10%.
```

Reply:
So I think this specifically, we cannot actually do yet. I think currently we literally only trigger on "data is added" and at that point there is no way for us to check what the data is. What we can do is use the filtering module to ensure data only gets added in certain conditions, like when an ml model detects something.
So how about:

```suggestion
You must configure data management to use Webhooks.
Webhooks will trigger every time data with the type binary, tabular, or file gets synced.
To limit when webhooks triggered for binary data, you can use the [`filtered-camera`](https://github.com/erh/filtered_camera) module. 
For a tutorial on filtering data, see [Selectively capture data using filtered-camera](/tutorials/projects/filtered-camera/).
```

Reply:
I wish it was possible to programmatically turn on data sync only when data of a certain value is captured.

For now, what should we put here?
```suggestion
For example, you can configure a webhook to send you a notification when your robot's sensor collects a new reading.
```

Reply:
Yeah I think that suggesiton is good @JessamyT . @nataliajacobowitz you happy with that?

Reply:
Happy with the above. We will iterate as new capabilities are supported with webhooks (hopefully soon)